### DISCUSSION TITLE
WebGL vendor isn't masked

### DISCUSSION THREAD

**Discussion by BornSaint** [Kaliiiiiiiiii-Vinyzu/patchright#18] [Ideas]
I have a situation about using mobile user agent, and everything is running well but WebGL vendor. 
If i use unset, it shows my offboard GPU.
If --disable-gpu is set, it shows Vulkan, which is equally bad masking since mobile will use mobile hardware as WebGL vendor.
Even this software doing very well bypassing sites which detect bots, this particular thing is not pruned yet, hope you add an option to set this programmatically, cheers

**Comment by Vinyzu** [comment]
This is out of scope for patchright.
